pdr_backend/analytics/check_network.py

```diff
@@ -179,26 +179,26 @@ def check_network_main(ppss: PPSS, lookback_hours: int):
 
     logger.info("Checking account balances")
 
-    OCEAN = web3_pp.OCEAN_Token
+    stake_token = web3_pp.stake_token
 
     addresses = get_opf_addresses(web3_pp.network)
     for name, address in addresses.items():
-        ocean_bal = OCEAN.balanceOf(address)
+        prediction_bal = stake_token.balanceOf(address)
         native_bal = Wei(web3_pp.get_token_balance(address))
 
         ocean_warning = (
-            " LOW OCEAN BALANCE!"
-            if ocean_bal < Eth(10).to_wei() and name != "trueval"
+            " LOW PREDICTION TOKEN BALANCE!"
+            if prediction_bal < Eth(10).to_wei() and name != "trueval"
```

Comment:
Why do we refere to the stake token as prediction token here? I suggest we keep it consistent with the rest and name is to stake_token_bal

Reply:
Good catch, I first changed it to that then made it prediction token, missed this one.

Fixed now!